tests/caasp/stack_update.pm

```diff
@@ -61,9 +61,11 @@ sub orchestrate_velum_reboot {
     my @tags = qw(velum-retry velum-bootstrap-done);
     assert_screen \@tags, $n * 900;
     if (match_has_tag 'velum-retry') {
-        record_soft_failure 'bsc#000000 - Should have passed first time';
+        record_soft_failure 'bsc#000000 - Update failed once, retrying';
```

Comment:
Hehe, finally we started to use magic `bsc#0` which was used before to trick OBS/IBS =).
Consider using `record_info` with `result` argument.

Reply:
record_info with `result` is broken (by intention maybe)
 - https://progress.opensuse.org/issues/17462

Reply:
Hmm, for me it looks like it works as intended, it was designed not to propagate failure to the job status, but just shown one square with it. But it doesn't serve your purpose.

Reply:
It worked like that before (it propagated jobs status) - what is the purpose of red squares if result is green?